CommandLineOptions.cpp

```diff
@@ -84,7 +100,9 @@ CommandLineOptions ParseCommandLineOptions(int argc, char **argv)
     CommandLineOptions result;
     for(int i = 1; i < argc; i++)
     {
-        ParseSingleCommandLineOption(argv[i], result);
+        std::string sanitized_option = argv[i];
+        Trim(sanitized_option);
+        ParseSingleCommandLineOption(sanitized_option, result);
```

Comment:
Okay so I noticed that this trims the option and then parses a single sanitized option. If you pass, say `-I/usr/local/include` then you're fine. You're also fine if you pass something like `-I=/usr/local/include` (as strange as that is). However, if you do `-I /usr/local/include` (which is fairly common) you might be in trouble. I would add some TODO somewhere marking that limitation.

Reply:
Done